src/index.ts

```diff
@@ -77,22 +77,44 @@ const writeData = async (content: Question) => {
 	// console.log();
 };
 
+//return user determined number of questions
+
+function returnNumberOfRandomQuestions<T>(questions: T[], n: number): any[] {
```

Comment:
Could the parameters be `(questions: Question[], n: number): Question[]` for more specificity? Would this work?

Otherwise LGTM